### PR TITLE
Compressor

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus-tests/build.gradle [hunk 1]
```diff
@@ -6,6 +6,10 @@ tasks.publish.enabled = false
 runStories.treatKnownIssuesOnlyAsPassed = true
 debugStories.treatKnownIssuesOnlyAsPassed = true
 
+configurations.all {
+    exclude group:"org.bouncycastle", module: "bcpkix-jdk15on"
+}
```
Comment (valfirst) [inline vividus-tests/build.gradle:11]: not for this PR

File: vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                throw new UncheckedIOException(e);
+            }
+        }
+        return image;
+    }
+
+    private BufferedImage toRGB(byte[] imageBytes) throws IOException
+    {
+        var image = ImageTool.toBufferedImage(imageBytes);
+        BufferedImage jpgImage = new BufferedImage(image.getWidth(), image.getHeight(), BufferedImage.TYPE_INT_RGB);
```
Comment (uarlouski) [inline vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java:63]: why TYPE_INT_RGB? :)

File: vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+{
+    private float imageCompressionQuality;
+
+    public byte[] toJpeg(byte[] image)
+    {
+        if (imageCompressionQuality < 1)
+        {
+            try (ByteArrayOutputStream baos = new ByteArrayOutputStream())
+            {
+                Iterator<ImageWriter> writers = ImageIO.getImageWritersBySuffix("jpg");
```
Comment (uarlouski) [inline vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java:43]: why did you choose jpg?

File: vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java (hunk 2 above)
Comment (ikalinin1) [reply vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java:63]: "Bogus input colorspace" is thrown if image has alpha channel

File: vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java (hunk 3 above)
Comment (ikalinin1) [reply vividus-extension-selenium/src/main/java/org/vividus/report/ui/ImageCompressor.java:43]: jpg is a png with the comression?

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**